src/views/datasets/buoy-telemetry/summary.vue

```diff
@@ -212,10 +212,12 @@ const generateQuery = (query) => {
 const endDate = store.maxDateRaw;
 const startDate = new Date(endDate.valueOf() - 5 * 24 * 60 * 60 * 1000);
 const monthDate = new Date(endDate.valueOf() - 30 * 24 * 60 * 60 * 1000);
+const twomonthDate = new Date(endDate.valueOf() - 60 * 24 * 60 * 60 * 1000);
+const threemonthDate = new Date(endDate.valueOf() - 90 * 24 * 60 * 60 * 1000);
```

Comment:
Bleh... it might be time for a date library. Try `date-fns`

```js
import subMonths from 'date-fns/subMonths'
const twomonthDate = subMonths(endDate, 2)
```

Reply:
thank you, this worked beautifully!